src/Form.jsx

```diff
@@ -1173,8 +1186,28 @@ function Form({
                         field={field}
                         fieldValues={fieldValues}
                         submit={submit}
-                        addRepeatedRow={addRepeatedRow}
-                        removeRepeatedRow={removeRepeatedRow}
+                        onClick={() => {
+                            let data;
+                            switch (field.link) {
+                                case 'add_repeated_row':
+                                    data = addRepeatedRow({});
+                                    break;
+                                case 'remove_repeated_row':
+                                    data = removeRepeatedRow({
+                                        index: field.repeat
+                                    });
+                                    break;
+                                default:
+                                    break;
+                            }
+                            if (data) {
```

Comment:
`data` will always be defined here - you probably want to check for `data.fieldKeys.length > 0` if you're checking if any servars changed